pom.xml

```diff
@@ -246,6 +247,13 @@
 		<mail.contentType>text/html</mail.contentType>
 
 		<mail.support.to>support@sigmah.org</mail.support.to>
+		
+		<!-- Optimisation email properties. -->
```

Comment:
Do we really need to have those in the pom.xml file? Maybe having those values directly in `sigmah.properties` could be OK because I don't see why we may have to change those while building the project. I think @mohamedkha just "did like above" but in the context it may not be needed.